collection/Collections.java

```diff
@@ -68,17 +68,18 @@
         return java.util.Collections.unmodifiableList(list);
     }
 
-    public static <T> List<T> list(Collection<T>... collections) {
-        List<T> combined = new ArrayList<>();
-        for (Collection<T> collection : collections) {
-            combined.addAll(collection);
-        }
+    public static <T> List<T> list(Collection<T> collection1, T item, T... array) {
+        List<T> combined = new ArrayList<>(collection1);
+        combined.add(item);
+        combined.addAll(Arrays.asList(array));
         return java.util.Collections.unmodifiableList(combined);
     }
 
-    public static <T> List<T> list(Collection<T> collection, T... array) {
+    public static <T> List<T> list(Collection<T> collection, Collection<T>... collections) {
```

Comment:
The signature needs one more `Collection<T>` so it does not clash with the single `list(Collection<T> collection)` @flyingsilverfin :
```
public static <T> List<T> list(Collection<T> collection1, Collection<T> collection2, Collection<T>... collections)
```